src/kubernetes/config.aat.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${SERVICE_NAME}-config
+  namespace: ${NAMESPACE}
+data:
+  IDAM_API_URL: https://preprod-idamapi.reform.hmcts.net:3511
+  IDAM_AUTHENTICATION_WEB_URL: https://idam.preprod.ccidam.reform.hmcts.net
+  IDAM_S2S_AUTH: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+  PAY_URL: http://payment-api-aat.service.core-compute-aat.internal
+  CLAIM_STORE_URL: http://cmc-claim-store-aat.service.core-compute-aat.internal
+  FEES_URL: http://fees-register-api-aat.service.core-compute-aat.internal
+  DRAFT_STORE_URL: http://draft-store-service-aat.service.core-compute-aat.internal
+  FEATURE_TOGGLES_API_URL: http://rpe-feature-toggle-api-aat.service.core-compute-aat.internal
+  POSTCODE_LOOKUP_API_KEY: ${POSTCODE_LOOKUP_API_KEY}
```

Comment:
this should move to deployment.template.yaml, its definition isn't env specific

Reply:
i moved the UV_THREADPOOL_SIZE too - to leave only aat specific vars